src/private/MDMPerformerGroup.m

```diff
@@ -72,6 +72,19 @@ - (void)executeLog:(MDMTransactionLog *)log trace:(MDMTrace *)trace {
     if ([performer respondsToSelector:@selector(addPlan:)]) {
       [(id<MDMPlanPerforming>)performer addPlan:plan];
     }
+    
+    // MDMNamedPlanPerforming callbacks
+    if (log.name.length) {
+      if (log.isRemoval) {
+        if ([performer respondsToSelector:@selector(removePlanNamed:)]) {
+          [(id<MDMNamedPlanPerforming>)performer removePlanNamed:log.name];
+        }
+      } else {
+        if ([performer respondsToSelector:@selector(addPlan:named:)]) {
+          [(id<MDMNamedPlanPerforming>)performer addPlan:plan named:log.name];
+        }
+      }
```

Comment:
We'll find it beneficial to keep track of which performer was associated with a given name. Otherwise, each time a named plan is added we'd have to ask every performer to remove any plan associated with `name` because we wouldn't know who has one.